src/index.js

```diff
@@ -197,21 +203,18 @@ export default function svelte(options = {}) {
 					})
 				);
 
-				if (css) {
-					// handle pre- and post-1.60 signature
-					const code = typeof compiled.css === 'string' ? compiled.css : compiled.css && compiled.css.code;
-					const map = compiled.css && compiled.css.map || compiled.cssMap;
+				if (css || options.emitCss) {
+					let fname = id.replace('.html', '.scss');
```

Comment:
should this be `.css` rather than `.scss`?

Reply:
Oops :) My use case bleeding through